### ISSUE TITLE
wasm-pack build --target bundler not including *_bg.js file in package.json

### ISSUE THREAD

**Issue by arlopurcell** [rustwasm/wasm-pack#837]
## 🐛 Bug description
When running `wasm-pack build --target bundler --scope scopename modulename`, there is are both *.js and *_bg.js files generated. the package.json file looks like this
`
{
  "name": "@scopename/modulename",
  "collaborators": [
    "arlo <arlo@purcell.live>"
  ],
  "version": "0.4.3",
  "files": [
    "modulename_bg.wasm",
    "modulename.js",
    "modulename.d.ts"
  ],
  "module": "modulename.js",
  "types": "modulename.d.ts",
  "sideEffects": false
}
`

#### 🤔 Expected Behavior
I think the `modulename.js` line in `files` should be `modulename_bg.js`, since `modulename.js` file will be included anyway because of the `"module": "modulename.js"` line. I'm not sure about that detail, but when i do `wasm-pack pack`, the generated tar ball is missing the `modulename_bg.js` file, so my module doesn't work

#### 👟 Steps to reproduce
`wasm-pack build --target bundler --scope scopename modulename`

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.9.1
rustc version: rustc 1.43.0 (4fb7144ed 2020-04-20)

**Comment by arlopurcell** [comment]
After testing this out locally, it looks like both `modulename.js` and `modulename_bg.js` should be in the files list

**Comment by tlaukkan** [comment]
Facing the same issue.

**Comment by Pauan** [comment]
@Alxandr I'm just waiting for @ashleygwilliams to add me as an owner on wasm-pack so I can do the publish.

**Comment by dbrgn** [comment]
Hit this as well, already created 2 broken releases on npm :confused: 

I hate to ping people, but since it's been over a month since the fix already: It would be awesome if this could be released on crates.io, @ashleygwilliams :slightly_smiling_face:

**Comment by prestwich** [comment]
For anyone coming here from google who wants a temporary solution, my current workaround is something like

```
$ git clone git@github.com:rustwasm/wasm-pack.git
$ cd wasm-pack
$ cargo run -- build --target bundler ../my_wasm_project
```

**Comment by pachi** [comment]
I think @ashleygwilliams is catching up and we'll probably have a new release not far away.

**Comment by kigawas** [comment]
Another option:

Use `cargo install --git https://github.com/rustwasm/wasm-pack.git` instead

**Comment by lucperkins** [comment]
Wait, why was this issue closed? This problem is still present in v0.9.1 (the current version) and bit me as well. I would strongly encourage the maintainers to address this.

**Comment by kigawas** [comment]
> Wait, why was this issue closed? This problem is still present in v0.9.1 (the current version) and bit me as well. I would strongly encourage the maintainers to address this.

It looks like repos under the rustwasm org are not actively maintained ;-p
